### PR TITLE
refactor(whiteboard): adjust styles

### PR DESCRIPTION
**Pull request by hyrious**

### CODE REVIEW

File: packages/flat-stores/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "white-web-sdk": ">=2.16"
   },
   "devDependencies": {
-    "@netless/fastboard": "1.0.0-canary.6",
+    "@netless/fastboard": "1.0.0-canary.7",
```
Comment (BlackHole1) [inline packages/flat-stores/package.json:17]: Split to another PR

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

File: packages/flat-stores/package.json (hunk 1 above)
Comment (hyrious) [reply packages/flat-stores/package.json:17]: This upgrade just updates some styles, so I guess it is safe to be included here.

**Review by BlackHole1 — APPROVED** [review]
